### ISSUE TITLE
Can't copy to clipboard

### ISSUE THREAD

**Issue by Hsn723** [n1k0/SublimeHighlight#74]
ST3, W10 1607. Removed and reinstalled SublimeHighlight multiple times, both from master and python3 tree.
```
Traceback (most recent call last):
  File "C:\Program Files\Sublime Text 3\sublime_plugin.py", line 812, in run_
    return self.run(edit, **args)
  File "C:\Users\natsu\AppData\Roaming\Sublime Text 3\Packages\SublimeHighlight\SublimeHighlight.py", line 117, in run
    pygmented = self.highlight(output_type, full)
  File "C:\Users\natsu\AppData\Roaming\Sublime Text 3\Packages\SublimeHighlight\SublimeHighlight.py", line 105, in highlight
    return pygments.highlight(self.code, self.get_lexer(),
  File "C:\Users\natsu\AppData\Roaming\Sublime Text 3\Packages\SublimeHighlight\SublimeHighlight.py", line 88, in get_lexer
    lexer = pygments.lexers.guess_lexer(code)
  File "C:\Users\natsu\AppData\Roaming\SUBLIM~1\Packages\pygments\all\pygments\lexers\__init__.py", line 252, in guess_lexer
    raise ClassNotFound('no lexer matching the text found')
pygments.util.ClassNotFound: no lexer matching the text found
```

**Comment by zcwang3** [comment]
same issue on Mac OS with latest Sublime and plugin

copy as RTF does not work when code is view with Java Syntax

Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 812, in run_
    return self.run(edit, **args)
  File "/Users/zcwang3/Library/Application Support/Sublime Text 3/Packages/Highlight/SublimeHighlight.py", line 117, in run
    pygmented = self.highlight(output_type, full)
  File "/Users/zcwang3/Library/Application Support/Sublime Text 3/Packages/Highlight/SublimeHighlight.py", line 105, in highlight
    return pygments.highlight(self.code, self.get_lexer(),
  File "/Users/zcwang3/Library/Application Support/Sublime Text 3/Packages/Highlight/SublimeHighlight.py", line 88, in get_lexer
    lexer = pygments.lexers.guess_lexer(code)
  File "/Users/zcwang3/Library/Application Support/Sublime Text 3/Packages/pygments/all/pygments/lexers/__init__.py", line 252, in guess_lexer
    raise ClassNotFound('no lexer matching the text found')

**Comment by n1k0** [comment]
Just a friendly reminder that this project is looking for a new maintainer :) see #78

**Comment by zcwang3** [comment]
😯

**Comment by zcwang3** [comment]
😯，so sad
do we have other plugin has similar functionality?

**Comment by n1k0** [comment]
Not any I'm aware of but I didn't look very hard as I'm not using Sublime anymore.

**Comment by zcwang3** [comment]
got it. Thanks very much 👍

**Comment by Hsn723** [comment]
For anyone still having this issue, I took a look at it today and it might be a regex bug with the `python3` branch. `guess_lexer_from_syntax` uses the regex `re.match(r"Packages/.*/(.*?)\.tmLanguage$", syntax)` (line 60) when it should be `re.match(r"Packages/.*/(.*?)\.(?:tmLanguage|sublime-syntax)$", syntax)` like on `master`. In fact that issue has already been fixed on `master` and the `python3` branch hasn't been updated in a year. I don't know if the `master` branch is compatible with python3, but I'm closing this issue since it works for me now after editing that line.
